examples/sample-application/server/server.go

```diff
@@ -129,12 +136,12 @@ func logoutHandler(deps *serverDeps) safehttp.Handler {
 	})
 }
 
-var invalidAuthErr responses.Error = responses.NewError(
-	safehttp.StatusBadRequest,
-	template.MustParseAndExecuteToHTML("Please specify a username and a password, both must be non-empty and your password must match the one you use to register."),
-)
-
 func postLoginHandler(deps *serverDeps) safehttp.Handler {
+	invalidAuthErr := responses.NewError(
```

Comment:
I moved it up in a different change. Probably because I mistakenly thought it was inside the handler and parsed on every request.